### ISSUE TITLE
Flow deprecation info into search service

### ISSUE THREAD

**Issue by xavierdecoster** [NuGet/NuGetGallery#7297]
Package deprecation info needs to flow into search service.
Required for phase 2 of package deprecation PMUI and more advanced search filtering.

**Comment by scottbommarito** [comment]
Likely not needed for MVP.

**Comment by skofman1** [comment]
removing out of the Epic, since it covers only the MVP.

**Comment by nkolev92** [comment]
Once this is fixed, we'd appreciate a heads up so we can fix the NuGet.Client side experience: https://github.com/NuGet/Home/issues/8719

**Comment by nkolev92** [comment]
This is a blocker for https://github.com/NuGet/Home/issues/8058.

**Comment by nkolev92** [comment]
@joelverhagen @loic-sharma 

Is this on any roadmap as of now? 
This would really help improve the performance of PM UI browse scenarios.

**Comment by joelverhagen** [comment]
Nope, not scheduled right now. Let's talk offline with @skofman1 about when this could happen w.r.t. to our other commitments.

**Comment by skofman1** [comment]
@nkolev92, since this is needed for an e2e experience (client perf improvement) we need to align priorities & schedules between the teams. i.e., how high is NuGet/Home#8058 on the client team backlog? Right now I see it marked as P3. We can discuss further offline.

**Comment by nkolev92** [comment]
Sounds good. I think the first target would be: https://github.com/NuGet/Home/issues/10865. 

I've started a discussion to prioritize that issue from a client perspective. 
I'll tag you in the chat for reference. 

Thanks @joelverhagen @skofman1!
